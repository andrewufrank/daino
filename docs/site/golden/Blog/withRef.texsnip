TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/withRef.md", dyLink = "Blog/withRef.md", dyLang = DLenglish, dyTitle = "postWithReference", dyAbstract = "What are the objects of ontologists .", dyAuthor = "FILL AUTHOR", dyDate = Just "2010-07-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Just "resources/chicago-fullnote-bibliography-bb.csl", dyReferences = Just (Array [Object (fromList [("author",Array [Object (fromList [("family",String "Fenner"),("given",String "Martin Beat")])]),("container-title",String "Nature Materials"),("id",String "fenner2012a"),("issued",Object (fromList [("month",String "3"),("year",String "2212")])),("page",String "261-263"),("publisher",String "Nature Publishing Group"),("title",String "One-click science marketing"),("type",String "article-journal"),("volume",String "11")])]), dyNoCite = Just "@navratil08", dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/withRef.md", link = "Blog/withRef.md", title = "postWithReference", abstract = "What are the objects of ontologists .", author = "FILL AUTHOR", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "An example post \\citep{fenner2012a}. With a reference to\n\\citep{frank09geo} given and navratil08 as nocite. Not including the\n\"TUxx9999\" addition in brackets. The \\citep{frank2010a} and\n\\citep{frank09geo} is a The references are in\n\"resources/BibTexLatex.bib\"."}