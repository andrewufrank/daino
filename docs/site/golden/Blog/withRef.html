<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="FILL AUTHOR" />
              <meta name="dcterms.date" content="2010-07-29" />
              <meta name="keywords" content="homepage" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">postWithReference</h1>
        
                <p class="author">FILL AUTHOR</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=Blog/withRef.html><em>postWithReference </em></a>
				    <br>What are the objects of ontologists .		     
				    <span class=tiny>FILL AUTHOR "2010-07-29" 
            Blog/withRef.html Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractWhat are the objects of ontologists . 
    content-from-panrep-blocks<p>An example post <span class="citation" data-cites="fenner2012a">[@fenner2012a]</span>. With a reference to <span class="citation" data-cites="frank09geo">[@frank09geo]</span> given and navratil08 as nocite. Not including the "TUxx9999" addition in brackets. The <span class="citation" data-cites="frank2010a">[@frank2010a]</span> and <span class="citation" data-cites="frank09geo">[@frank09geo]</span> is a The references are in "resources/BibTexLatex.bib".</p>
<h1 id="references">References</h1>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
